changelogs/fragments/79399-resolvelib_lt_0_10_0.yml

```diff
@@ -0,0 +1,2 @@
+major_changes:
+  - Upgrade `resolvelib >= 0.5.3, < 0.10.0`
```

Comment:
@s-hertel does this good enough?

Reply:
Didn't the reviewer request "minor" changes, where you used "major"?

Reply:
Sorry my miss, typo mistake, fixed and force pushed.